app/src/test/ActionCreatorTestHelpers.ts

```diff
@@ -1,12 +1,33 @@
 import {createMockStore} from "./mocks/mockStore";
 import {expect} from "chai";
 
+
+export interface VerifyActionThatCallsServiceProperties {
+    mockServices: () => void;
+    callActionCreator: () => void;
+    expectTheseActions: any[];
+}
+
 export interface VerifyActionThatCallsServiceAndReturnsResultProperties {
     mockServices: () => void;
     callActionCreator: () => void;
     expectTheseActionTypes: any[];
 }
 
+export function verifyActionThatCallsService(
```

Comment:
I'm on board with introducing some more test helpers, it is nicer, you're right. I added a more general helper that allows you to check the full actions, not just the action types because in the case I was testing, the data returned by the service isn't returned with the action.